### PR TITLE
Travel Better - Hunter

### PR DESCRIPTION
**Pull request by elliottcrush**
#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

…-hx versioning

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/hunter/src/less/affix.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//== Affix
```
Comment (stuartgiles) [inline brands/hunter/src/less/affix.less]: Remove? See hilton?

File: brands/hunter/src/less/type.less [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+.feature-heading {
+  font-size: ceil((@font-size-default * 5)); // ~75px
+  line-height: ceil((@font-size-default * 5.6)); // ~84px
+  margin: 0;
+  * + & {
+    margin-top: ceil((@font-size-default * 5)) // ~75px
+  }
+}
+
+h1,
```
Comment (stuartgiles) [inline brands/hunter/src/less/type.less]: Remove extras class names, instead use a comment.

File: brands/hunter/src/less/backgrounds.less [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+  .bg-brand .bg-muted {
+    background-color: @bg-muted-inverse-normal;
+  }
+}
+// Inverse backgrounds
+.bg-primary {
+  .bg-inverse;
+  background-color: @bg-primary;
+  #gradient > .vertical(@start-color: @bg-primary, @end-color: darken(@bg-primary, 8%));
```
Comment (stuartgiles) [inline brands/hunter/src/less/backgrounds.less]: remove

File: brands/hunter/src/less/buttons.less [hunk 4]
```diff
@@ -0,0 +1,81 @@
+@import '../../../brand-globals/src/less/buttons';
+
+//
+// Button styles
+// --------------------------------------------------
+
+.hx-btn-ultimate {
+  .button-variant(@btn-primary-color; @btn-primary-bg; @btn-primary-border);
```
Comment (stuartgiles) [inline brands/hunter/src/less/buttons.less]: Trial just using .btn-primary

File: brands/hunter/src/less/colours.less [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+
+
+
+
+
+
+// Product colors
```
Comment (stuartgiles) [inline brands/hunter/src/less/colours.less]: Remove? Until needed

File: brands/hunter/src/less/mixins/buttons.less [hunk 6]
```diff
@@ -0,0 +1,71 @@
+// Button variants
```
Comment (stuartgiles) [inline brands/hunter/src/less/mixins/buttons.less]: Remove and go back to a tweak of bootstraps default blur

File: brands/hunter/src/less/mixins/panels.less [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// Panels
+
+.panel-variant(@border; @heading-text-color; @heading-bg-color; @heading-border) {
+
+  & > .panel-heading {
```
Comment (stuartgiles) [inline brands/hunter/src/less/mixins/panels.less]: Add border top to the `.panel-body` (not when its following a `.panel-header`).

Then, make the `.panel-primary .panel-header` a solid purple background

File: brands/hunter/src/less/scaffolding.less [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+// * {
+//   outline: 1px dotted rgba(0,0,0,0.1);
+// }
+
+
+// Reset
+
+html {
+  font-size: 15px;
```
Comment (stuartgiles) [inline brands/hunter/src/less/scaffolding.less]: variable

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request merged**